pyiron/vasp/base.py

```diff
@@ -777,14 +771,14 @@ def write_magmoms(self):
         else:
             s.logger.debug("No magnetic moments")
 
-    def set_eddrmm_handling(self, status="not_converged"):
+    def set_eddrmm_handling(self, status="warn"):
         """
         Sets the way, how EDDRMM warning is handled.
 
         Args:
-            status (str): new status of EDDRMM handling (can be 'not_converged', 'ignore', or 'restart')
+            status (str): new status of EDDRMM handling (can be 'warning', 'ignore', or 'restart')
```

Comment:
should it be `"warn"` instead of `"warning"`?

Reply:
Yes!